test/v1/users.js

```diff
@@ -44,4 +44,23 @@ describe('/v1/users endpoints', () => {
           testFor(2, {id: 2, name: 'Vasanth'}, done);
       });
   });
+
+
```

Comment:
May be we should do this. For now, we could just have like this , as each test is fairly simple